x-pack/plugins/canvas/public/state/reducers/resolved_args.test.js

```diff
@@ -41,13 +40,15 @@ describe('resolved args reducer', () => {
       });
 
       const newState = resolvedArgsReducer(state, action);
-      expect(newState.resolvedArgs['element-1']).to.eql([
-        {
-          state: 'pending',
-          value: null,
-          error: null,
-        },
-      ]);
+      expect(newState.resolvedArgs['element-1']).toMatchInlineSnapshot(`
+        Object {
+          "0": Object {
```

Comment:
This is the return value produced by `resolvedArgsReducer()` which used to work by the check that `.to.eql()` ran. Jest assertions are a bit more picky (in a good way, the return value shouldn't eql an array) but I just wanted to raise awareness a little... this is weirdness